lib/timeline/Core.js

```diff
@@ -241,15 +257,12 @@ Core.prototype._create = function (container) {
     }
   }
 
-  if (this.dom.centerContainer.addEventListener) {
-    // IE9, Chrome, Safari, Opera
-    this.dom.centerContainer.addEventListener("mousewheel", onMouseWheel.bind(this), false);
-    // Firefox
-    this.dom.centerContainer.addEventListener("DOMMouseScroll", onMouseWheel.bind(this), false);
-  } else {
-    // IE 6/7/8
-    this.dom.centerContainer.attachEvent("onmousewheel", onMouseWheel.bind(this));
-  }
+  // Add modern wheel event listener
+  const wheelType = "onwheel" in document.createElement("div") ? "wheel" : // Modern browsers support "wheel"
+    document.onmousewheel !== undefined ? "mousewheel" : // Webkit and IE support at least "mousewheel"
+    this.dom.centerContainer.addEventListener ? "DOMMouseScroll" : // Older Firefox versions like "DOMMouseScroll"
+    "onmousewheel" // All the rest should be comfortable with "onmousewheel"
+  this.dom.centerContainer.addEventListener(wheelType, onMouseWheel.bind(this), false);
```

Comment:
Use ``attachEvent`` here in case ``addEventListener`` is not supported (IE < 9).

Reply:
That's right. Thank you!